src/applications/burials/components/NoFormPage.jsx

```diff
@@ -62,6 +63,31 @@ const renderFields = [
   },
 ];
 
+const determineDeathCertificate = formData => {
+  if (environment.isLocalhost()) {
+    return formData?.transportationReceipts?.length > 0
```

Comment:
curious if this is a problem with our mockdata not being in sync with the API call. can you walk me through this tomorrow?